tests/base.py

```diff
@@ -529,3 +539,84 @@ def pre_run(self, rpsns, rqps_num):
         self.rqps_num = rqps_num
         self.rpsns = rpsns
         self.to_rts()
+
+
+class Mlx5DcResources(TrafficResources):
```

Comment:
Wouldn't it be better to keep it in a separate file (e.g. base_mlx5 or base_dc?)

Reply:
I'll consider doing so (base_mlx5) in v2.
I thought of separating provider-specific tests into: tests/providers/mlx5 tests/providers/efa etc.. then each provider can have its own utils/base if needed. But that may come later, not as part of this series

Reply:
Makes sense, we plan to push EFA traffic tests soon too.

Reply:
I think that this provider(s) test(s)/resource(s) need to be in a specific provider file.

Reply:
> I think that this provider(s) test(s)/resource(s) need to be in a specific provider file.

I agree. As I said, will do it in a separate series.